Extracting Individual Colour channels/Image Augmentation in Python

```diff
@@ -0,0 +1,147 @@
+**1** Loading Image into Environment
+
+Importing required Libraries
+"""
+
+import cv2
+import urllib.request
+import numpy as np
+from google.colab.patches import cv2_imshow
```

Comment:
Is this module required? It is only specific for usage in Google colab. :)

Reply:
It is you're right. I kept it incase somebody wanted to copy paste and see it.
Should I remove it instead?

Reply:
Oh alright. Then it's fine. 👍🏻

Reply:
👍🏻, Do add the tag incase of being admin/maintainer.